autonomy/chain/mint.py

```diff
@@ -118,10 +117,19 @@ def verify_and_fetch_token_id_from_event(
 
 def get_contract(public_id: PublicId) -> Contract:
     """Load contract for given public id."""
-    contract_dir = RELATIVE_CONTRACTS_DIR / public_id.name
+
+    # check if a local package is available
+    contract_dir = CONTRACTS_DIR_LOCAL / public_id.name
     if contract_dir.exists():
-        return contract_dir
-    return ROOT_DIR / contract_dir
```

Comment:
This method should return an instance of a contract not a path to the contract. idk how the mypy missed this 🤔